src/InlineText.php

```diff
@@ -7,4 +7,34 @@
  */
 class InlineText extends Text
 {
+
+    public function __construct($text = "", $markdownTag = "")
```

Comment:
There shouldn't be a default value here. (see comment further down about rebase)